main.go

```diff
@@ -777,8 +791,9 @@ func (s *Server) printJSONReport(w http.ResponseWriter, req *http.Request) {
 		w.WriteHeader(http.StatusNotFound)
 		w.Write(errMsgBytes)
 		return
+	} else {
+		api.PrintJSONReport(w, req, s.analyzers[release].Report, s.analyzers[release+"-prev"].Report, s.options.EndDay, 15)
```

Comment:
Seems like you should be able to have a single PrintJSONReport function whether you are reporting a single release, or multiple.  Printing a single release report is just a special case of printing multiple and i'd expect the output to look the same (other than only having a single release entry)